apps/atqueue/values.yaml

```diff
@@ -17,6 +17,11 @@ env: {}
 envSecrets: []
 # -- This key allows specification of a script to override the entrypoint
 entrypoint:
+# -- This key allows for specification of Butler secret information
```

Comment:
It looks like you mix sentences ending in a `.` (like line 25) and not ending in a `.` (like this line). Please choose one and stick to that.

Reply:
Same for the other `values.yaml` files.

Reply:
Ugh. Didn't even follow my own conventions. Fixed.